automated-testing/e2e-testing/testing-comparison.md

```diff
@@ -0,0 +1,14 @@
+# Unit vs Integration vs System vs E2E Testing
+
+To better understand which testing methodology when to apply it in your project, the below table illustrates the most critical characteristics and differences between Unit, Integration, System and End-to-End testing:
+
+| Unit Test | Integration Test | System Testing | E2E Test |
```

Comment:
It seems like we should have a left hand title for what each row is actually comparing.

Reply:
Added